lib/services/network/downloads_service.dart

```diff
@@ -110,20 +111,13 @@ class AttachmentDownloadController extends GetxController {
 
       error.value = true;
       attachmentDownloader._removeFromQueue(this);
-      return Response(requestOptions: RequestOptions(path: ''));
+      return null;
     });
-    if (response.statusCode != 200) return;
-    Uint8List bytes;
-    if (attachment.mimeType == "image/gif") {
-      bytes = await fixSpeedyGifs(response.data);
-    } else {
-      bytes = response.data;
-    }
-    if (!kIsWeb && !kIsDesktop) {
```

Comment:
The fixSpeedyGifs stuff was moved to downloadAttachment. Granted, this whole attachment code is very messy and should be tested on all the platforms